src/services/motion/motion.ts

```diff
@@ -1,4 +1,6 @@
+import { GeoObstacle } from '../../gen/common/v1/common_pb';
```

Comment:
instead of passing the raw proto, let's use the `AsObject` object - we should add the following definition to custom types (`src/types.ts`) and use that:

```ts
// src/types.ts

export type GeoObstactle = common.GeoObstactle.AsObject;
```